votes/initiateNewVote/_EDIT_ME.yml

```diff
@@ -3,7 +3,7 @@
 
 # 1. Select a subject for the vote. This can be a question addressed to the TSC
 #    voting members.
-subject: REPLACEME
+subject: What UX do we want `node` to handle non-existent `.env` files passed to its CLI?
```

Comment:
I don't have a suggestion but something in this question seems off, linguistically.

Reply:
Yes, I agree. To me it should be "How should `node` handle non-existent `.env` files passed via CLI"

Reply:
```suggestion
subject: How should `node` handle non-existent `.env` files passed via CLI flags?
```